aiohttp/streams.py

```diff
@@ -328,6 +328,9 @@ def end_http_chunk_receiving(self) -> None:
             if not_enough:
                 await self._wait('readline')
 
+        # fixes memory leak: https://github.com/aio-libs/aiohttp/issues/3631
+        self._http_chunk_splits = []
```

Comment:
@asvetlov  let me deal with this bug

Reply:
https://github.com/aio-libs/aiohttp/pull/3635 it is here. @mvanderkroon good catch! Thanks for sample code. Sorry that I forced you to create PR. Anyway it helped a lot to understand what is the source of the problem.

Reply:
@socketpair, happy to help!